src/Moq/It.cs

```diff
@@ -234,6 +258,29 @@ public static TValue IsIn<TValue>(IEnumerable<TValue> items)
 			return Match<TValue>.Create(value => items.Contains(value), () => It.IsIn(items));
 		}
 
+		/// <summary>
+		///   Matches any value that is present in the sequence specified.
+		/// </summary>
+		/// <param name="items">The sequence of possible values.</param>
+		/// <param name="comparer">An <see cref="IEqualityComparer{T}"/> with which the values should be compared with.</param>
+		/// <typeparam name="TValue">Type of the argument to check.</typeparam>
+		/// <example>
+		///   The following example shows how to expect a method call with an integer argument
+		///   with value from a list.
+		///   <code>
+		///     var values = new List&lt;int&gt; { 1, 2, 3 };
+		///
+		///     mock.Setup(x => x.HasInventory(
+		///                         It.IsAny&lt;string&gt;(),
+		///                         It.IsIn(values)))
+		///         .Returns(false);
+		///   </code>
+		/// </example>
```

Comment:
This (copy-and-pasted, I suppose) example doesn't really fit, as it doesn't make use of a `IEqualityComparer<int>` at all. Please either replace the example, or remove it altogether; I'm fine with both.